Source/WebCore/rendering/style/StyleRareInheritedData.h

```diff
@@ -151,6 +151,7 @@ class StyleRareInheritedData : public RefCounted<StyleRareInheritedData> {
     unsigned textAlignLast : 3; // TextAlignLast
     unsigned textJustify : 2; // TextJustify
     unsigned textDecorationSkipInk : 2; // TextDecorationSkipInk
+    unsigned mathStyle : 1; // MathStyle
```

Comment:
What’s the motivation for this change?

Reply:
Just better bit-packing, I could have put my new bit in this extra spot, but my new bit went better with the ones below.

Reply:
I split out this change to https://github.com/WebKit/WebKit/pull/47222